kubernetes/reference/adoc/IHV/HPE/ProLiant.adoc

```diff
@@ -1,15 +1,32 @@
 
-==== {familyHPE-DL-URL}[{an_HPE} {familyHPE-DL} DL Rack Servers]
+==== link:{familyHPE-DL-URL}[{an_HPE} {familyHPE-DL} DL Rack Servers]
```

Comment:
Just a stupid question: Is it a good idea to have links in headers? This could be a possible cause for the conversion to DocBook to fail.

Reply:
> Just a stupid question: Is it a good idea to have links in headers? This could be a possible cause for the conversion to DocBook to fail.

For now, have done this in all RCs to make an obvious link to the respective h/w platforms. Of course I can change this, so for the next RC set, I'll try the other way (no link in header, but in first text content instead).  And if that looks okay, will adjust all the RCs to that approach.

Reply:
So - if you have used the link iin the header also for those documents that are already published (Dell Poweredge and Supermicro Superserver), there should not be an issue with the conversion to DocBook. But after having had a quick look at the Dell and Supermicro docs, it seems that you have used it before. It works well on the HTML pages, but the header does not link in the PDF versions.

Reply:
Good point (as I mostly just review PDFs, but don't use them as much). I saw the color being the same, so just assumed they were links.

With that said, for the next RC iteration setting up Cisco, I'll remove header link and put it in the text space and with that pull request, will do the same for all the other RCs as well ... if that is okay.

Reply:
Ok, great. Thanks for the clarification. :+1: